tests/cli/test_compose_ext4-filesystem.sh

```diff
@@ -16,8 +16,21 @@ set -e
 
 CLI="${CLI:-./src/bin/composer-cli}"
 
-
 rlJournalStart
+    rlPhaseStartSetup
+        if [[ -n $SUBSCRIPTION_HOSTNAME && -n $SUBSCRIPTION_USERNAME && -n $SUBSCRIPTION_PASSWORD ]]; then
+            rlRun "mkdir /etc/yum.repos.d/disabled"
+            rlRun "mv /etc/yum.repos.d/*.repo /etc/yum.repos.d/disabled"
+            rlRun "subscription-manager config --server.hostname=$SUBSCRIPTION_HOSTNAME"
+            rlRun "subscription-manager register --username=$SUBSCRIPTION_USERNAME --password=$SUBSCRIPTION_PASSWORD"
```

Comment:
This is exposing the username & password in the logs. See current log. Run the command without rlRun and examine the exit code to figure out if it succeeded or not.